### PR TITLE
Fix `input_ray_pickable` documentation for `CollisionObject3D`

### PR DESCRIPTION
**Pull request by rafallus**
Mainly taken from the `CollisionObject2D` docs. The previous description was incorrect.

### CODE REVIEW

File: doc/classes/CollisionObject3D.xml [hunk 1]
```diff
@@ -183,7 +183,7 @@
 			If [code]true[/code], the [CollisionObject3D] will continue to receive input events as the mouse is dragged across its shapes.
 		</member>
 		<member name="input_ray_pickable" type="bool" setter="set_ray_pickable" getter="is_ray_pickable" default="true">
-			If [code]true[/code], the [CollisionObject3D]'s shapes will respond to [RayCast3D]s.
+			If [code]true[/code], this object is pickable. A pickable object can detect the mouse pointer entering/leaving, and if the mouse is inside it, report input events. Requires at least one [code]collision_layer[/code] bit to be set.
```
Comment (akien-mga) [inline doc/classes/CollisionObject3D.xml:186]:
```suggestion
			If [code]true[/code], this object is pickable. A pickable object can detect the mouse pointer entering/leaving, and if the mouse is inside it, report input events. Requires at least one [member collision_layer] bit to be set.
```
Should also be changed in 2D if it's the same there.

Comment (rafallus) [reply doc/classes/CollisionObject3D.xml]: Yeah, changed in 2D too

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.5.

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.3.

**Pull request merged**